### DISCUSSION TITLE
Tycho 3.0 release date

### DISCUSSION THREAD

**Discussion by akurtakov** [eclipse-tycho/tycho#1158] [General]
It's time to set the date for 3.0 release . Right now I'm thinking of end of September so there is enough time to finish what's started, accomodate Eclipse 4.25 bundles (due on September 14, 2022 ) and etc.
WDYT?

**Comment by laeubi** [comment]
Sounds good, I think the relevant parts are already done see https://github.com/eclipse/tycho/issues?q=is%3Aopen+is%3Aissue+milestone%3A3.0

One thing that might be worth to finish is the new resolver stuff, but I'm also fine to have it explicitly enabled for now, and the sisu cleanup.
